rest/data/project_settings.go

```diff
@@ -75,10 +75,15 @@ func CopyProject(ctx context.Context, env evergreen.Environment, opts CopyProjec
 	if err := event.CopyProjectSubscriptions(oldId, projectToCopy.Id); err != nil {
 		catcher.Wrapf(err, "copying subscriptions from project '%s'", oldIdentifier)
 	}
-	// set the same admin roles from the old project on the newly copied project.
+	// Set the same admin roles from the old project on the newly copied project.
 	if err := model.UpdateAdminRoles(projectToCopy, projectToCopy.Admins, nil); err != nil {
 		catcher.Wrapf(err, "updating admins for project '%s'", opts.NewProjectIdentifier)
 	}
+
+	// We can pass in an empty before here since the project is new
```

Comment:
Can you please elaborate on this comment (an empty what)?

Reply:
Before is the name of the project settings variable. I can add that.

Reply:
thanks!